smoke-tests/smoke-e2e.bats

```diff
@@ -17,6 +17,11 @@ teardown_file() {
   assert_equal "$result" '"test-span"'
 }
 
+@test "SDK can send metrics" {
+  result=$(metric_names_for ${SMOKE_TEST_SCOPE})
+  assert_equal "$result" '"smoke-test.int.metric"'
```

Comment:
I'm confused about how this test passes. You added a button send the metrics, but you haven't updated the SmokeTests code to actually click the button. Are you clearing the data.json before you run the smoke-tests?

Reply:
That is a great question. Even weirder is how does it pass in CI??

Reply:
Are we not running smoke tests in CI?

Reply:
‼️ Looks like my last PR (#10) broke the distro, smoke tests on main no longer pass.

Debug mode now redirects spans to the console instead of sending them to BOTH console and normal exporter :sob: